intake_stac/catalog.py

```diff
@@ -152,6 +153,25 @@ def _load(self):
     def _get_metadata(self, **kwargs):
         return kwargs
 
+    def to_geopandas(self):
+        """
+        Load the STAC Item Collection into a geopandas GeoDataFrame
+        """
+        try:
+            import geopandas as gpd
+        except ImportError:
+            raise ImportError(
+                "Using to_geopandas requires the `geopandas` package"
+            )
```

Comment:
I think the drop in test coverage is coming from this block. Either add a #noqa here or try this mock approach: https://medium.com/python-pandemonium/how-to-test-your-imports-1461c1113be1

Reply:
Thank you, @jhamman